enrolment-service/enrol/core/views.py

```diff
@@ -1,11 +1,11 @@
+# For email dev testing, use 'python -m smtpd -n -c DebuggingServer localhost:1025'
```

Comment:
Little goodie to help dev time

Reply:
Maybe add a note about this in the README for developers

Reply:
good idea! will change that right now

Reply:
moved...